libs/prisma-value/src/sql_ext.rs

```diff
@@ -15,6 +15,7 @@ impl<'a> From<ParameterizedValue<'a>> for PrismaValue {
             ParameterizedValue::Uuid(uuid) => PrismaValue::Uuid(uuid),
             ParameterizedValue::DateTime(dt) => PrismaValue::DateTime(dt),
             ParameterizedValue::Char(c) => PrismaValue::String(c.to_string()),
+            ParameterizedValue::Bytes(_) => todo!(),
```

Comment:
I'll try to get the types mapping PR merged today asap. But afaik this match branch is never reached.